nestkernel/spike_data.h

```diff
@@ -128,9 +136,13 @@ class SpikeData
   double get_offset() const;
 };
 
+//!< check legal size
```

Comment:
Doxygen documentation placed before the thing they document should start with `//!` only.